### ISSUE TITLE
Is my training loss normal?

### ISSUE THREAD

**Issue by Guzaiwang** [zylo117/Yet-Another-EfficientDet-Pytorch#30]
Thanks for your great job to re-implement the efficient-det in Pytorch.

**Comment by Guzaiwang** [comment]
On my custom dataset, my training loss is 
Step: 24. Epoch: 0/20. Iteration: 25/1412. Cls loss: 24130.16602. Reg loss: 0.52574. Total loss: 24130.69141:   2%| | 25/1412 

I doubt that such a loss is not normal.  Could you show us your training logs such as loss?

**Comment by zylo117** [comment]
Did you load pretrained weights?
And it's only 24 steps...
Just wait.

**Comment by Guzaiwang** [comment]
> Did you load pretrained weights?
> And it's only 24 steps...
> Just wait.

Thanks，the pretrain weights should be loaded correctly except the last classification layers for finetuning.

**Comment by Guzaiwang** [comment]
I changed the lr from 1e-5 to 1e-3. The training loss reduced to 10~ and it works normally.

**Comment by zylo117** [comment]
sorry for the training troubles, there's a bug in loss function. please pull the latest code and give it a try.
